cli/utils.py

```diff
@@ -195,6 +201,7 @@ def string_to_list(value, separator=',', append_to_list=True):
             value = [value, ]
     return value
```

Comment:
This methods works with dictionaries, which are mutable objects. There is no need to return the object you're passing into the method, someone may think this method return a new dict object...